recipes/tls-securepair-to-tlssocket/tests/input/basic-test-file-3.js

```diff
@@ -0,0 +1,13 @@
+const tls = require('node:tls');
+const fs = require('fs'); // Code non lié
+
+function createSecureConnection() {
+  // Using tls.SecurePair constructor
+  const pair = new tls.SecurePair();
+  
+  // Ces lignes doivent disparaître
```

Comment:
nit: remove or translate it to english please?

```suggestion
  // Ces lignes doivent disparaître
```